modin/pandas/test/dataframe/test_default.py

```diff
@@ -214,11 +213,36 @@ def test_combine_first():
     df_equals(modin_df1.combine_first(modin_df2), pandas_df1.combine_first(pandas_df2))
 
 
+def test_corr():
+    eval_general(
+        *create_test_dfs(test_data["int_data"]),
+        lambda df: df.corr(),
+    )
+    # AssertionError is arisen because Modin result may slightly differ
+    # from pandas result due to floating pointing arithmetic.
+    try:
+        eval_general(
+            *create_test_dfs(test_data["float_nan_data"]),
+            lambda df: df.corr(),
```

Comment:
Why do you test functions with only default parameters?

Reply:
Because `method` is set to `pearson` by default. For all rest values of `method` it is defaulting to pandas.

Reply:
I think we could expand testing parameters for `min_periods`.